### ISSUE TITLE
Handling of missing close tags in Glimmer vs Angular, etc.?

### ISSUE THREAD

**Issue by brodybits** [prettier/prettier#11170]
This may be similar to #11169.

**Prettier 2.3.2**
[Playground link](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEACVAeAJgSwG4B8AOlOtvsaepjgLYDmqAzgE5gC8RIAhgA68A6AFa96XStWoYA9LkIgANCAi8YOaE2ShuLFhADuABR0JNKbgBt93AJ6alAIxbcwAazgwAyrxc4o9ZBgWAFc4JThaBzgsLGiAGW5-YO56OAAxCBZabhg1f2QeYJgIRRAACxhaCwB1Mpx4Jh8wOE9Tevx6mwKwJnsQPyY4FhhDZ3ps5AAzS0GlISYADwAhZzcPT25aODi-OCmZsJB5hc8-egs4AEVgiHh9i1mQHxZBlgLzui23pV4WPxhqjgsDAysgABwABh+ekG1WcvAKvzgrzweyUAEcbvBRiozDwmABaKBwaLRUosOCYnAU0YpCZIaYPQ6DWg4QIhZlnC7XW57BkHJQwbgOQHA0FIABMgucOAsZwAwhBaPSQMiAKylYKDAAqwrMjMeeFCAEkoLFYJ4wH9VABBM2eGA2C73QYAX1dQA)

```sh
--parser glimmer
```

**Input:**

```hbs
  <div>
  <div>
    <img src="app.jpg">
      </div>
```

**Output:**

```hbs
SyntaxError: Unclosed element `div`: 

|
|  <div>
|

(error occurred in 'an unknown module' @ line 1 : column 2) (1:2)
> 1 |   <div>
    |  ^^^^^
  2 |   <div>
  3 |     <img src="app.jpg">
  4 |       </div>
```

**Expected behavior:**

Add missing `</div>` tag?

Here is what I get with `angular` parser, for example:

**Prettier 2.3.2**
[Playground link](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEACVAeAJgSwG4B8AOlOtvsaepjgLYDmqAzgE5gC8RIAhgA68A6AFa96XStWoYA9LkIgANCAi8YOaE2ShuLFhADuABR0JNKbgBt93AJ6alAIxbcwAazgwAyrxc4o9ZBgWAFc4JThaBzgsLGiAGW5-YO56OAAxCBZabhg1f2QeYJgIRRAACxhaCwB1Mpx4Jh8wOE9Tevx6mwKwJnsQPyY4FhhDZ3ps5AAzS0GlISYADwAhZzcPT25aODi-OCmZsJB5hc8-egs4AEVgiHh9i1mQHxZBlgLE+mCLHVLeFj8YNUcFgYGVkAAOAAMSj+EEG1WcvAKfzgrzweyUAEcbvBRiozDwmABaKBwaLRUosODYnBU0YpCZIaYPQ6DWg4QIhVlnC7XW57JkHJQwbgOIEgsFIABMwucOAsZwAwhBaIyQKiAKylYKDAAqorMzMeeFCAEkoLFYJ4wP9VABBC2eGA2C73QYAX3dQA)

```sh
--parser angular
```

**Input:**

```html
  <div>
  <div>
    <img src="app.jpg">
      </div>
```

**Output:**

```html
<div>
  <div>
    <img src="app.jpg" />
  </div>
</div>

```

Similar results with `html` & `lwc` parsers.

And with `vue` parser (similar to <https://github.com/prettier/prettier/issues/11169#issuecomment-874416247>):

**Prettier 2.3.2**
[Playground link](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEACVAeAJgSwG4B8AOlOtvsaepjgLYDmqAzgE5gC8RIAhgA68A6AFa96XStWoYA9LkIgANCAi8YOaE2ShuLFhADuABR0JNKbgBt93AJ6alAIxbcwAazgwAyrxc4o9ZBgWAFc4JThaBzgsLGiAGW5-YO56OAAxCBZabhg1f2QeYJgIRRAACxhaCwB1Mpx4Jh8wOE9Tevx6mwKwJnsQPyY4FhhDZ3ps5AAzS0GlISYADwAhZzcPT25aODi-OCmZsJB5hc8-egs4AEVgiHh9i1mQHxZBlgK8UNLeFj8YapwsDAysgABwABiU3wgg2qzl4BW+cFeeD2SgAjjd4KMVGYeEwALRQODRaKlFhwDE4cmjFITJDTB6HQa0HCBEJMs4Xa63Pb0g5KGDcBz-QHApAAJgFzhwFjOAGEILQ6SAkQBWUrBQYAFSFZgZjw+cAAklBYrBPGAfqoAIKmzwwGwXe6DAC+LqAA)

```sh
--parser vue
```

**Input:**

```vue
  <div>
  <div>
    <img src="app.jpg">
      </div>
```

**Output:**

```vue
<div>
  <div>
    <img src="app.jpg">
      </div>

```

**Comment by fisker** [comment]
I don't think we should, the missing close tags in HTML are actually valid HTML code.

**Comment by lydell** [comment]
Related: https://github.com/prettier/prettier/issues/5864
